rerun_cpp/src/rerun/recording_stream.cpp

```diff
@@ -256,4 +256,33 @@ namespace rerun {
 
         return status;
     }
+
+    Error RecordingStream::try_log_file_from_path(std::string_view filepath) const {
+        if (!is_enabled()) {
+            return Error::ok();
+        }
+
+        rr_error status = {};
+        rr_recording_stream_log_file_from_path(_id, detail::to_rr_string(filepath), &status);
+
+        return status;
+    }
+
+    Error RecordingStream::try_log_file_from_contents(
+        std::string_view filepath, std::string_view contents
```

Comment:
as mentioned already, I find `std::string_view` for raw bytes to be extremly confusing. In C++17 the only clear way to pass an array of bytes I'm aware of is `std::byte* data, size_t data_size` :(